### PR TITLE
fix(issue-7478): resolving missing modules & versions

### LINKED ISSUE #7478: appengine.flexible.websockets.main_test: test_websocket failed
Note: #5819 was also for this test, but it was closed more than 10 days ago. So, I didn't mark it flaky.

----

commit: 881e0b2bd9c37e4f7019bc5207555811142318c0
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/05477c79-b47a-4a00-94ea-847efc341133), [Sponge](http://sponge2/05477c79-b47a-4a00-94ea-847efc341133)
status: failed
<details><summary>Test output</summary><br><pre>Traceback (most recent call last):
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/urllib3/connection.py", line 175, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/urllib3/util/connection.py", line 95, in create_connection
    raise err
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/urllib3/util/connection.py", line 85, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/urllib3/connectionpool.py", line 710, in urlopen
    chunked=chunked,
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/urllib3/connectionpool.py", line 398, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/urllib3/connection.py", line 239, in request
    super(HTTPConnection, self).request(method, url, body=body, headers=headers)
  File "/usr/local/lib/python3.7/http/client.py", line 1281, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/local/lib/python3.7/http/client.py", line 1327, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/usr/local/lib/python3.7/http/client.py", line 1276, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/usr/local/lib/python3.7/http/client.py", line 1036, in _send_output
    self.send(msg)
  File "/usr/local/lib/python3.7/http/client.py", line 976, in send
    self.connect()
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/urllib3/connection.py", line 205, in connect
    conn = self._new_conn()
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/urllib3/connection.py", line 187, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x7ff2b65fb750>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/requests/adapters.py", line 450, in send
    timeout=timeout
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/urllib3/connectionpool.py", line 786, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=37873): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff2b65fb750>: Failed to establish a new connection: [Errno 111] Connection refused'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/workspace/appengine/flexible/websockets/main_test.py", line 53, in server
    check_server('http://{}/'.format(bind_to))
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/retrying.py", line 49, in wrapped_f
    return Retrying(*dargs, **dkw).call(f, *args, **kw)
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/retrying.py", line 212, in call
    raise attempt.get()
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/retrying.py", line 247, in get
    six.reraise(self.value[0], self.value[1], self.value[2])
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/six.py", line 719, in reraise
    raise value
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/retrying.py", line 200, in call
    attempt = Attempt(fn(*args, **kwargs), attempt_number, False)
  File "/workspace/appengine/flexible/websockets/main_test.py", line 51, in check_server
    requests.get(url)
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/requests/sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/requests/sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "/workspace/appengine/flexible/websockets/.nox/py-3-7/lib/python3.7/site-packages/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=37873): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff2b65fb750>: Failed to establish a new connection: [Errno 111] Connection refused'))</pre></details>

### PR DESCRIPTION
**Pull request by pattishin**
## Description

Fixes #7478 

Fusion: https://source.cloud.google.com/results/invocations/dd66b6e0-62fb-40c6-b05a-1301c18ebbbd/targets

*Steps to re-create error:*

Using `appengine` as an example, but should apply to other affected samples.

1) Use locally or cloud shell and navigate to the sample folder and simply install requirements
```
pip3 install -r requirements.txt
pip3 install -r requirements-test.txt
```

2) You should see the same error logs like so after executing the following:
```
python3 main.py
```
Error:
Screen Shot 2022-03-10 at 9 47 45 PM

3) Verify this PR no longer produces those missing module errors and for `appengine` - verify the tests are passing locally.

Screen Shot 2022-03-10 at 9 48 27 PM

Note: You'll see varying versions for `markupsafe` are being installed. This is due to the varying `Flask` versions per sample
ref: https://github.com/pallets/flask/issues/4455#issuecomment-1044223329

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [x] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

File: composer/airflow_1_samples/requirements.txt [hunk 1]
```diff
@@ -4,6 +4,7 @@ apache-airflow[gcp]==1.10.15
 apache-airflow-backport-providers-google==2021.3.3
 apache-airflow-backport-providers-cncf-kubernetes==2021.3.3
 kubernetes==11.0.0
+markupsafe==2.0.1
```
Comment (dandhlee) [inline composer/airflow_1_samples/requirements.txt:7]: Composer one's already been taken care of! No need to add it.

**Review by dandhlee — CHANGES_REQUESTED** [review]
Thank you! One minor change.

File: composer/airflow_1_samples/requirements.txt (hunk 1 above)
Comment (pattishin) [reply composer/airflow_1_samples/requirements.txt]: Sounds good!

**Review by dandhlee — APPROVED** [review]
Thank you!

**Comment by dandhlee** [comment]
Addressing dependency issues, admin merging.

**Pull request merged**